data/faq/nl.yml

```diff
@@ -43,41 +46,80 @@
+        - Als een school de eerste ronde in eigen huis organiseert, moeten alle deelnemers van die school de eerste ronde op school afleggen. Een deel van de leerlingen op school en een ander deel in een regionaal centrum is niet mogelijk.<br>
+        - Alle deelnemers aan de Informatica-olympiade van uw school die behoren tot eenzelfde categorie, beginnen op hetzelfde moment aan hun proef (niet enkelen om 14u en anderen om 15u).<br>
     -
       question: Is er een oplossing voor de overlap met een andere wedstrijd?
       answer: |
-        In 2016 valt de eerste ronde van de Informatica-olympiade samen met die van de Biologie-olympiade. De biologie-olympiade gaat door op school in een computerklas van 13u tot 15u. Uw leerlingen kunnen na afloop de Informatica-olympiade afleggen (pas beginnend om 15u of net iets later als er een kleine pauze is), onder de volgende bijkomende voorwaarden:<br>
-        - De deelnemers hebben vanaf 14u30 geen contact met de buitenwereld: scholieren geven gsm, smartwatches en andere apparaten met internetverbinding af, met uitzondering van wat nodig is om de biologie-olympiade te beëindigen.<br>
-        - Alle leerlingen die deelnemen aan de Informatica-olympiade beginnen op hetzelfde moment (niet enkelen om 14u en anderen om 15u).<br>
-        
-        Wij vinden deze overlap met een andere wedstrijd zelf jammer, maar ze was niet te vermijden. Ze is een gevolg van het feit dat we als nationale olympiade rekening moeten houden met de verschillende kalenders van andere wedstrijden in zowel Vlaanderen als in Franstalig België. 
+        In 2017 valt de eerste ronde van de Informatica-olympiade samen met die van de Biologie-olympiade. De biologie-olympiade gaat ook door op school in een computerklas, op vaste tijdstippen. Uw leerlingen mogen voordien of na afloop de Informatica-olympiade afleggen.<br>
+        Wij vinden deze overlap met een andere wedstrijd zelf jammer, maar ze is niet te vermijden. Ze is een gevolg van de langere voorbereidingstijd die we nodig hebben in 2017, en het feit dat we als nationale olympiade rekening moeten houden met de kalenders van andere wedstrijden in zowel Vlaanderen als in Franstalig België. 
+    -
+      question: Is een computerklas noodzakelijk?
+      answer: De eerste ronde is in 2017 gepland als een wedstrijd via de computer. Indien de voorbereidingen daarvoor succesvol zijn, zal de wedstrijd in een computerklas met internetverbinding moeten plaatsvinden.
     -
-      question: Moet er aan de computer gewerkt worden in de eerste ronde ?
-      answer: "Nee, de eerste ronde is enkel een papieren examen."
+      question: Mijn computerklas is niet groot genoeg!
+      answer: |
+        Het zal zeer waarschijnlijk toegelaten worden de eerste ronde in maximaal 3 aparte sessies te organiseren, m.n. 1 sessie per deelnemerscategorie. 
```

Comment:
Let's not, after years of carefully monitoring potential cheating, suddenly go all wild wild west and allow just anything. Some consistency would be nice. I've rephrased it to make separate sessions possible, and added a line about exceptions to the rule if there are some guarantees, but I think we should still recommend a default setting: that everyone from the same school and same grade, starts at the same time in the same location.